code/particle/util/ParticleProperties.cpp

```diff
@@ -62,6 +62,21 @@ void ParticleProperties::parse(bool nocreate) {
 			Warning(LOCATION, "Could not find curve '%s'", buf.c_str());
 		}
 	}
+
+	if (optional_string("+Rotation:")) {
+		SCP_string buf;
+		stuff_string(buf, F_NAME);
+		if (!stricmp(buf.c_str(), "DEFAULT")) {
+			m_rotation_type = RotationType::DEFAULT;
+		} else if (!stricmp(buf.c_str(), "RANDOM")) {
+			m_rotation_type = RotationType::RANDOM;
+		} else if (!stricmp(buf.c_str(), "FIXED")) {
+			m_rotation_type = RotationType::FIXED;
```

Comment:
I'm wondering if this might be called "screen aligned" or something instead, since "fixed" might be a bit vague. One might expect that to mean fixed with regard to some global reference frame instead of the viewer's point of view!

Reply:
Agreed, and I'm certainly open to other terminology.  @MoerasGrizzly does `SCREEN_ALIGNED` work?

Reply:
screen_aligned definitely works.

Reply:
Ok, modified